doc-tool/resources/_layouts/api-page.html

```diff
@@ -169,21 +171,27 @@ <h1 class="section {% if entity.hasVisibleMembers == false %}empty{% endif %}">
                     </div><!-- end member-title -->
 
                     <div class="member-body">
-                        <div class="member-body-short" id="short-body-{{ member.signature }}">
-                            {{ member.comment.short }}
-                        </div>
                         <div class="member-body-long" id ="long-body-{{ member.signature }}">
                             {{ member.comment.body }}
                         </div>
                     </div>
                     <script>
+
+                        function toggleConstructorsBody(span, sig){
+                            var members = document.getElementById("entity-constructors");
+                            members.classList.toggle("toggled");
+                            span.innerHTML = (span.innerHTML == "[-]") ?  "[+]" : "[-]";
+                        }
+
+                        function toggleMembersBody(span, sig){
+                            var members = document.getElementById("entity-members");
+                            members.classList.toggle("toggled");
+                            span.innerHTML = (span.innerHTML == "[-]") ?  "[+]" : "[-]";
+                        }
```

Comment:
This should be the same function - but perhaps taking the `members` variable as argument.